Add public documentation for Infrastructure Package Catalog in CSM VM (#26294)
DataDog/documentation: content/en/security/cloud_security_management/vulnerabilities/_index.md

{{< img src="security/vulnerabilities/csm_remediate.png" alt="Details explorer of a specific vulnerability highlighting the ability to remediate and assign to team member" width="100%">}}

## Explore infrastructure packages

> The [Infrastructure Packages Catalog][10] provides a real-time inventory of all packages across hosts, host images and container images deployed in your infrastructure, offering an interface to investigate your SBOMs, enriched with vulnerability and runtime context.

Reviewer: ```suggestion
The [Infrastructure Packages Catalog][10] provides a real-time inventory of all packages across hosts, host images, and container images deployed in your infrastructure. It offers an interface you can use to investigate your SBOMs, enriched with vulnerability and runtime context.
```

> Quickly assess the impact of a critical emerging vulnerability by searching for affected package versions and identifying all resources using it.

Reviewer: ```suggestion
You can assess the impact of a critical emerging vulnerability by searching for affected package versions and identifying all of the resources that use it.
```

Author: took into account the suggestion
